src/pylorax/api/queue.py

```diff
@@ -141,18 +141,22 @@ def queue_sort(uuid):
             # The anaconda logs are also copied into ./anaconda/ in this directory
             os.makedirs(joinpaths(dst, "logs"), exist_ok=True)
 
-            loggers = (log, program_log, dnf_log)
-            log_file_names = ("composer.log", "program.log", "dnf.log")
-            
-            handlers = []
-            for logger, log_file_name in zip(loggers, log_file_names):
+            def make_handler(log_file_name):
                 handler = logging.FileHandler(joinpaths(dst, "logs", log_file_name))
                 handler.setLevel(logging.DEBUG)
                 handler.setFormatter(logging.Formatter("%(asctime)s %(levelname)s: %(message)s"))
-                logger.addHandler(handler)
+                return handler
+
+            loggers = (log, program_log, dnf_log)
```

Comment:
You could just make this a list of tuples and then unpack them instead of using zip() lower down. That would make the relationship between the elements clearer.

Reply:
Yeah, 083f60b should make this a bit cleaner.